index.html

```diff
@@ -562,21 +562,38 @@
 \begin{eqnarray}
 S_\textrm{diffuse}    = \mathrm{Slab}(f_\mathrm{diffuse})
 \end{eqnarray}
-where $f_\mathrm{diffuse}$ is defined to be the classic Oren-Nayar model ([#Oren1994], [#Pharr2023]), which is a microfacet model where the microfacets are individually Lambertian, with albedo specified by **`base_weight`** multiplied by **`base_color`**, and a microfacet roughness parameter in $[0,1]$ given by **`base_roughness`**.
+where $f_\mathrm{diffuse}$ is defined to be the BRDF of the classic Oren-Nayar microfacet model ([#Oren1994], [#Pharr2023]) in which the microfacets are individually Lambertian. The resulting BRDF of the glossy-diffuse slab is the combination of a "glossy" specular lobe provided by reflection from the dielectric interface, and a diffuse lobe provided by bounces off the substrate. This models for example the reflection from shiny, totally opaque surfaces such as dense plastic, rock, and concrete.
 
-The resulting BRDF of the glossy-diffuse slab is the combination of a "glossy" specular lobe provided by reflection from the dielectric interface, and a diffuse lobe provided by bounces off the substrate. This models for example the reflection from shiny, totally opaque surfaces such as dense plastic, rock, and concrete. This form of BRDF model is classic in computer graphics and typically termed the Ashikhmin-Shirley or Kelemen model ([#Ashikhmin2000], [#Kelemen2001], [#Kulla2017], [#Kutz2021]).
+Physically there will be darkening and saturation of the observed color due to multiple inter-reflections (scattering) within the gloss layer, as described in the coat darkening section. However in this case, we wish to automatically compensate for this effect by adjusting the albedo of $f_\mathrm{diffuse}$ in order to make the observed color match the input color $\mathbf{C} =$ **`base_weight`** * **`base_color`** (in a similar spirit to the albedo remapping described in the Subsurface section).
```

Comment:
I'm assuming the coat darkening section is not something that can be linked to? (No link is produced for this section reference.)

Reply:
It will be linked to eventually, just it's still in review. The method we use to define the coloring of:
   - coat
  - glossy-diffuse
  - subsurface

are all using the same basic formulation, so we should make sure to connect the discussions.

Reply:
Makes sense and sounds good.